CTOR-1779-plugin-apps-monitoring-quanta-restapi-rework-current-plugin (#4639)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-monitoring-quanta-restapi.md

| Service Alias           | Service Template                                            | Service Description                      | Discovery  |
|:------------------------|:------------------------------------------------------------|:-----------------------------------------|:----------:|
| User Journey Incidents  | App-Monitoring-Quanta-Userjourney-Incidents-Restapi-custom  | Check incidents for a given user journey | X          |
| User Journey Statistics | App-Monitoring-Quanta-Userjourney-Statistics-Restapi-custom | Check statistics for a user journey      | X          |

> The services listed above are not created automatically when a host template is applied. To use them, [create a service manually](/docs/monitoring/basic-objects/services), then apply the service template you want.

> If **Discovery** is checked, it means a service discovery rule exists for this service template.

</TabItem>
</Tabs>

### Discovery rules

#### Service discovery

| Rule name                                             | Description                                            |
|:------------------------------------------------------|:-------------------------------------------------------|

> | App-Monitoring-Quanta-Restapi-Userjourneys-Incidents  | Discover user journeys of a site to monitor incidents  |

Reviewer: ```suggestion
| App-Monitoring-Quanta-Restapi-Userjourneys-Incidents  | Discover user journeys for a site and monitor incidents  |
```

> | App-Monitoring-Quanta-Restapi-Userjourneys-Statistics | Discover user journeys on a site to monitor statistics |

Reviewer: ```suggestion
| App-Monitoring-Quanta-Restapi-Userjourneys-Statistics | Discover user journeys for a site and monitor statistics |
```

<TabItem value="User Journey Statistics" label="User Journey Statistics">

| Name                                            | Unit  |
|:------------------------------------------------|:------|
| journey-performance-score                       | N/A   |
| journey-performance-score                       | N/A   |
| *metrics1*#journey.herotime.milliseconds        | ms    |
| *metrics2*#journey.herotime.milliseconds        | ms    |
| *metrics1*#journey.speedindex.time.milliseconds | ms    |
| *metrics2*#journey.speedindex.time.milliseconds | ms    |
| *metrics1*#journey.ttfb.milliseconds            | ms    |
| *metrics2*#journey.ttfb.milliseconds            | ms    |
| interaction-performance-score                   | N/A   |
| interaction-performance-score                   | N/A   |
| *metrics1*#herotime.milliseconds                | ms    |
| *metrics2*#herotime.milliseconds                | ms    |
| *metrics1*#speedindex.time.milliseconds         | ms    |
| *metrics2*#speedindex.time.milliseconds         | ms    |
| *metrics1*#ttfb.milliseconds                    | ms    |
| *metrics2*#ttfb.milliseconds                    | ms    |

</TabItem>
</Tabs>

## Prerequisites

> In order to monitor your Quanta interface, it is necessary to [configure your Quanta interface](https://app.quanta.io/app/settings/organizations).

Reviewer: ```suggestion
In order to monitor Quanta, you must have [a valid API token](https://quantaio.notion.site/Automatically-track-events-0ba555d180e145b4bc2e116327fe8a2d#:~:text=Authentication%20and%20token%20generation), and [have user journeys configured in Quanta](https://quantaio.notion.site/Creation-of-a-scenario-or-User-Journey-bdd229fbe6d14ac885e9a3e91a8d57f1).
Data from Real User Monitoring will only be accessible if you have completed [the integration of Quanta's RUM TAG](https://quantaio.notion.site/Installer-le-Real-User-Monitoring-fa29c4e7154540579d91d32719616546).
```
